### ISSUE TITLE
Getting Started with CSS

### ISSUE THREAD

**Issue by holygrimm** [Grimm-s-Alchemy-Chamber/hackers_hive#6]
#Title Of Blog Post : Getting Started with CSS:
###Short Description of what is expected : An introductory tutorial on Cascading Style Sheets (CSS) for styling HTML documents, covering selectors, properties, and basic styling techniques.

###Category : beginner's guide
###Tags : CSS, Web Development, Front-End

**Comment by seabeePraveen** [comment]
please assign this task to me

**Comment by himanchal08** [comment]
i would like to work on this issue it seems doable
i will try my best to implement it

**Comment by VIBER1** [comment]
yeah sure you can work on it and please refer to contributing guidelines.
